operator/pkg/ingress/envoy_config.go

```diff
@@ -392,8 +392,35 @@ func getRouteMatch(ingressPath slim_networkingv1.HTTPIngressPath) *envoy_config_
 	return nil
 }
 
+type SortableRoute []*envoy_config_route_v3.Route
+
+func (s SortableRoute) Len() int {
+	return len(s)
+}
+
+func (s SortableRoute) Less(i, j int) bool {
+	// Make sure Exact Match always comes first
+	isExactMatch1 := len(s[i].Match.GetPath()) != 0
+	isExactMatch2 := len(s[j].Match.GetPath()) != 0
+	if isExactMatch1 {
+		return true
```

Comment:
Should we also sort exact matches by length, or does it not matter?

e.g. should `/foobar` be before `/foo`?

Reply:
the exactmatch order didn't matter, as envoy will only come to one match consistently :)